Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -47,22 +39,23 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            if (array != null && array.Length != 0)
+            if (array == null || array.Length == 0)
             {
-                var arrayLength = array.Length;
-                // make a copy of array by value (not reference)
-                int[] tmp = new int[128];
-                for (var i = 0; i < arrayLength; i++)
-                {
-                    tmp[i] = array[i];
-                }
+                return;
+            }
 
-                // reverse current index with last index reduced with current index
-                for (var i = 0; i < arrayLength; i++)
-                {
-                    //Console.WriteLine("Swapping: " + array[i] + " and " + tmp[arrayLength - i-1]);
-                    array[i] = tmp[arrayLength - i-1];
-                }
+            var arrayLength = array.Length;
+            // make a copy of array by value (not reference)
+            int[] tmp = new int[128];
```

Comment:
Why 128? This should have the same length as the original array.

Reply:
You do not need this temp array anyway. You can solve this without creating a new array at all.